server/src/main/java/org/apache/iotdb/db/service/IoTDB.java

```diff
@@ -106,6 +106,10 @@ private void setUp() throws StartupException {
 
     Runtime.getRuntime().addShutdownHook(new IoTDBShutdownHook());
     setUncaughtExceptionHandler();
```

Comment:
maybe rpc is enable, but mtree is not ready, so user send a request will call npe exception?